### DISCUSSION TITLE
command line go installation does not work

### DISCUSSION THREAD

**Discussion by mikepruett3** [alecthomas/chroma#1018] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When I try to run the following

`go install github.com/alecthomas/chroma/v2`

I get errors, and cannot continue. I would like to be able to use go install vs cloning the repo and manually doing it.

### To Reproduce

I get the following error on windows:

```
go: 'go install' requires a version when current directory is not in a module
        Try 'go install github.com/alecthomas/chroma/v2@latest' to install the latest version
```

I try adding the @latest to the end of the command, and get this error:

`package github.com/alecthomas/chroma/v2 is not a main package`

Not sure how to proceed.
Trying from Windows 10 using go version go1.22.4 windows/amd64

**Comment by alecthomas** [comment]
Use `go get` to add modules to your project, not `go install`.

**Reply by mikepruett3** [reply]
When you try and use `go get` it gives an error, and tells you to use `go install` instead.

```bash
'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```
